src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,7 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @password_changable = !@configuration['change_password'] && !CONFIG['proxy_auth_mode'] && !CONFIG[:ldap_mode]
```

Comment:
It works since it's not set to `:off` even if you set it to. It only accepts :on and any other value is nil. First I matched against `== :off` but then it failed so I inspected it and found out it's `nil`. We had the same thought ;-)